napari/utils/interactions.py

```diff
@@ -210,15 +210,37 @@ def hello_world(layer, event):
     'Enter': '↵',
 }
 
-
+joinchar = '-'
 if sys.platform.startswith('darwin'):
     KEY_SYMBOLS.update(
         {'Control': '⌘', 'Alt': '⌥', 'Option': '⌥', 'Meta': '⌃'}
     )
+    joinchar = ''
```

Comment:
Default valid Qt shortcut sequences (and in other software in general) use + as the joinchar, so is this something we want to change?

Reply:
This it used only to set the tooltip values in UI. So that should be ok

Reply:
Yes but when we add customization of shortcuts, the user puts a new one, then when we ask Qt for the representation of that shortcut, it will return something that uses `+` as the separator, so we should probably keep using that t be in sync with what [Qt will provide](https://doc.qt.io/qt-5/qkeysequence.html).

Reply:
Yes but when we add customization of shortcuts, the user puts a new one, then when we ask Qt for the representation of that shortcut, it will return something that uses `+` as the separator, so we should probably keep using that t be in sync with what [Qt will provide](https://doc.qt.io/qt-5/qkeysequence.html).

Reply:
... this is related to my other comment about putting the shortcut in the `QAction` in the menu.  It'd be great if we can standardize on `+`, but it may break other things (like with vispy's naming?) ... anyway, some conversion func between the two might be necessary.

Reply:
I think this has been solved by https://github.com/napari/napari/pull/2554 (So thios comment could be collapsed :). )